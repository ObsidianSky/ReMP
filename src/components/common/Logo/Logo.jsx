import React from 'react';
if (process.env.BROWSER) {
	require('./Logo.scss');
}

const Logo = () => <div className="logo">ex.netflixroulette</div>;

export default Logo;