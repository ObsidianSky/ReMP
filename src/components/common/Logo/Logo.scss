@import '../../../assets/styles/variables';

.logo {
  font-size: 20px;
  color: $pink;
  display: inline-block;
}