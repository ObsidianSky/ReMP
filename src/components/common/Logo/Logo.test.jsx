import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Logo from './Logo';

describe('Logo', () => {

    test('should renders correctly', () => {
        const wrapper = shallow(<Logo />);

        expect(toJson(wrapper)).toMatchSnapshot();
    });
});