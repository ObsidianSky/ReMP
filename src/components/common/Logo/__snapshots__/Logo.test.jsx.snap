// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should renders correctly 1`] = `
<div
  className="logo"
>
  ex.netflixroulette
</div>
`;
